.github/workflows/compliance.yml

```
@@ -0,0 +1,114 @@
+#####################################################################################################################
+# @author       Michael Smith
+# @date         2021-08-23
+# @description  As SFDO Managed Packages start using the newly introduced Instrumentation Services to 'log' 
+#               usage data to Splunk and Argus, this Compliance Check action does two things:
+#               1. Ensures that Packaged Apex Classes do not make any direct reference to the core SfdoLogUtils class.
+#                  Any code references to this class will cause the Compliance build to fail.
+#               2. At least in the short term, adds an Instrumentation reviewer to any PR that in any way references or
+#                  or uses the SfdoInstrumentationService. This is primarily meant as a short term action to aid in
+#                  adoption of the new instrumentation services.
+#####################################################################################################################
+name: Instrumentation_Service_Compliance_Verification
+on:
+  pull_request:
+    ## Filter the job to only execute if CLS files are in the PR
+    paths:    
+      - '**.cls'
+      - '!unpackaged/**.cls'
+jobs:
+  Code_Check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Check for previously assigned Instrumentation Label
+        id: label_check
+        run: |
+          echo "::set-output name=prlabels::${{ join(github.event.pull_request.labels.*.name, ',') }}"
+
+      - name: Check for references to the SfdoLogUtils or SfdoInstrumentationServices Apex classes
+        # If the Instrumentation label is already assigned to the PR, all checks are skipped to avoid redundant PR comments and alerts
+        id: get_pr_diff
+        if: contains(steps.label_check.outputs.prlabels, 'Instrumentation') == false
```

You can use contains directly here:
```suggestion
        if: !contains(github.event.issue.labels.*.name, 'Instrumentation')
```

---

@jstvz  I could not get `!contains()` to not throw an error. For some reason it just didn't recognize that syntax.

---

Annoying. Did you try `!(contains(...))`?

Not a blocker.